### DISCUSSION TITLE
How can we use Pest for tests?

### DISCUSSION THREAD

**Discussion by ekandreas** [magic-test/magic-test-laravel#28] [Q&A]
The package enforces the use of phpunit but we want to use the awesome Pest testing tool with Magic.
Would it be possible to set the command `php artisan magic` to use ./vendor/bin/pest instead of phpunit?

**Comment by ekandreas** [comment]
https://pestphp.com/

**Reply by mateusjatenee** [reply]
@ekandreas it's possible, maybe it can accept an argument? 
generating pest tests is another matter tho

**Comment by ekandreas** [comment]
Still no progress? Pest is a really great tool when it comes to testing in Laravel...
